### PR TITLE
(BOLT-229) update to use new test helpers in BoltSpec and task_helpers

### PR DESCRIPTION
**Pull request by adreyer**
This depends on the follow PRs

https://github.com/puppetlabs/bolt/pull/549
https://github.com/puppetlabs/beaker-task_helper/pull/30

### CODE REVIEW

File: task_spec/spec/acceptance/init_spec.rb [hunk 1]
```diff
@@ -1,203 +1,45 @@
...
-        expect(res['status']).to eq('success')
-      end
-    end
-
-  end
+  include Beaker::TaskHelper::Inventory
+  include BoltSpec::Run
 
   def module_path
     RSpec.configuration.module_path
```
Comment (MikaelSmith) [inline task_spec/spec/acceptance/init_spec.rb]: This feels like something we should maybe put in a helper?

File: task_spec/spec/spec_helper_acceptance.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
-require 'puppet'
```
Comment (MikaelSmith) [inline task_spec/spec/spec_helper_acceptance.rb]: We'll be able to go back to this when we unvendor puppet, right?

File: task_spec/spec/acceptance/init_spec.rb (hunk 1 above)
Comment (adreyer) [reply task_spec/spec/acceptance/init_spec.rb]: This is rspec specific so it doesn't seem like it belongs in beaker-task_helpers. I'm not sure it's the best way to track this either.

File: task_spec/spec/spec_helper_acceptance.rb (hunk 2 above)
Comment (adreyer) [reply task_spec/spec/spec_helper_acceptance.rb]: yes

Comment (MikaelSmith) [reply task_spec/spec/spec_helper_acceptance.rb]: I think a comment reminding us of that would be nice.

**Pull request merged**